neps/nep-0568.md

```diff
@@ -187,18 +197,39 @@ In this NEP, we propose updating the ShardId semantics to allow for arbitrary id
 
 ## Reference Implementation
 
-```text
-[This technical section is required for Protocol proposals but optional for other categories. A draft implementation should demonstrate a minimal implementation that assists in understanding or implementing this proposal. Explain the design in sufficient detail that:
+```
```

Comment:
Can you add a short comment to each block? The pseudo code is nice but I think it would be good to also have something in human words ;)

Reply:
And please also add a subsection heading if necessary?